<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QWidget" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:checked
{
    background-color: #3daee9;
}

</string>
  </property>
  <widget class="QPushButton" name="printPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="showCompensateurPushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Compensateur</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="showCollimateurPushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Collimateur</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="showTreatmentPushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Seances</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="showControleDosePushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Controle de la dose</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="showDebitMesurePushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Débit mesuré</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="showControleQualitePushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Controle qualité</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="showSOBPPushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>SOBP</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
